src/auth/authService.ts

```diff
@@ -85,12 +85,13 @@ export class AuthService {
 
   public async login(path = '/'): Promise<void> {
     try {
-      return this.userManager.signinRedirect({ data: { path } });
+      await this.userManager.signinRedirect({ data: { path } });
     } catch (error) {
       if (error.message !== 'Network Error') {
         Sentry.captureException(error);
       }
-      return Promise.reject();
+    } finally {
+      Promise.resolve();
```

Comment:
If error occurs, previously a rejected Promise was returned, now a resolved Promise. That changes the behavior of this function. Is that ok?

Reply:
login() causes a full page redirect and none of the callers listen to the result. But returned value should of course be logical. I made some changes to catch a possible error and return rejected / resolved promise in a simpler way.